skimage/io/_io.py

```diff
@@ -113,6 +113,8 @@ def imsave(fname, arr, plugin=None, **plugin_args):
         Target filename.
     arr : ndarray of shape (M,N) or (M,N,3) or (M,N,4)
         Image data.
+    check_contrast : bool
```

Comment:
should be: `check_contrast : bool, optional`

Reply:
i guess the other params dont specify optional, but maybe you can add optional to them?